scripts/fix.sh

```diff
@@ -15,8 +15,12 @@ source ./scripts/updatebom.sh
 GO_CMD="go"
 
 function mod_tidy_fix {
-  run rm ./go.sum
-  run ${GO_CMD} mod tidy || return 2
+  local modules=("$@")
+  for module in "${modules[@]}"; do
+    local module_dir
+    module_dir="${module%/...}"
+    run_for_module "${module_dir}" run "${GO_CMD}" mod tidy
+  done
```

Comment:
`mod_tidy_fix` is supposed to run on a single module, and the loop is the responsibility of the caller.

Do not get time to read through the whole PR yet, is it possible not to change such logic to make the review easier? If this is a change unrelated to go workspace, then we should do it separately.